### ISSUE TITLE
[openssl 3] megasync needs rebuild

### ISSUE THREAD

**Issue by CyrusYip** [archlinuxcn/repo#3022]
### 问题类型 / Type of issues

* 其它 / other

### 受影响的软件包 / Affected packages

* megasync

----

请在此补充说明。
Please describe in detail here.

error message:

```
❯ megasync
megasync: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
```

**Comment by hubutui** [comment]
see also https://github.com/archlinuxcn/repo/issues/3014

**Comment by CyrusYip** [comment]
This issue was solved by megasync 4.7.2.0-1 .
